### DISCUSSION TITLE
Dall'SDK con furore

### DISCUSSION THREAD

**Discussion by crazyLillo** [tesivo/tesivo_s_VTMB_italian_translation_for_Unofficial_Patch#62] [Idee]
Usare notepad++  non è che sia proprio agevole per editare sia i file DLG che i file LIP.

per i file DLG vi ho fatto vedere che nell'SDK esiste un programmino apposito per localizzare, (una bomba, perfetto!)
se per caso c'è un file corrotto, ti dice dove è l'errore che si ripara facilmente.

Per i file LIP invece ci sono due programmini (Caption Editor) e (LipVcd Editor) che servono per editare i sottotitoli.

uno dei due (associato ad altre utillity) serve anche per modificare o creare nuovo audio e sottotitoli 
(si può modificare anche il labiale, la gestura, le espressioni facciali, tutto!
Dice che si potrebbe cambiare la lingua in modo integrale in tutto e per tutto (lavoraccio immane lunghissimo, MA si può FARE!!!)
Discorso che ci interessa fino a un certo punto dato l'enorme carico di lavoro da eseguire (doppiaggio integrale + labiale abbinato al parlato, le gestura e le espressioni facciali e sopratutto sincronizzazione) ma come ho già detto si può fare! I strumenti ci sono!

Ritornando ai sottotitoli, con "Caption Editor" (è un exe che una volta fatto partire la prima volta, insinua una voce di associazione file nel menù contestuale del tasto destro del mouse). la voce è "EDIT Subtitles".
ma una volta associato, non c'è bisogno di fare click con il destro, ma basta fare doppio click sx sul file lip in questione, aprendo il file senza problemi con il programmino apposito per i sottotitoli.

Ti si aprono un paio di finestrelle (una con sfondo nero, da non toccare) e un altra con i sottotitoli in inglese esistenti.

Basta copiare la traduzione al posto della frase in inglese e il gioco è FATTO!

Scordatevi di impazzire come ho fatto io, a contare i caratteri (anche con software adatti, perdi solo tempo).

Con il "programmino" basta inserire la traduzione, a contare i caratteri e inserire a fianco di "char" il numero corretto dei caratteri ci pensa lui in automatico. EASY PEASY  LEMON SQUEEZY !!!

L'ho capito solo ieri notte tardi, e i prossimi file lip li farò con quell'utility.

**Comment by tesivo** [comment]
A dire il vero io uso Pycharm, ma più che altro per tener traccia delle modifiche e per caricarle qui su GitHub ;) 

Niente male niente male, è chiaro che tutti i futuri lavori conviene siano svolti con questi 3 tools.

Fantastico!

**Reply by crazyLillo** [reply]
sto postando un blocco di altri 170 e piu file tutti tradotti con l'utility per i sottotitoli.
ho scoperto che se per sbaglio inserisco una lettera accentata e salvo, 
la lettera accentata viene trasformata in una lettera senza accenti!

**Reply by tesivo** [reply]
Niente male, immagino che occorra solo aggiungere a mano il carattere "'".

**Reply by crazyLillo** [reply]
il mio modo di operare adesso è questo :

traduco gli originali rispettando punteggiatura etc...
e metto tutto su blocco note.

vado nella cartella di lavoro con i file (copiati da originale).
apro ogni file con doppio click (si apre l'utility), inserisco la traduzione con un semplice copia e incolla da 

blocco note. e clicco su "Submit". che salva la modifica.

(cerco di fare le cose giuste a monte, senza andare a cercare pagliuzze poi alla deriva...)

rispondendo alla tua domanda, si se fosse necessario, anche se inserisci una lettera accentata, il sistema non 

la prende e mette al suo posto la stessa lettera senza accento e li si dovrebbe intervenire riaprendo il file e 

inserendo manualmente (l'apostrofo\accento).

...meglio correggere prima...che dopo...ma le sviste capitano a tutti...
